pnpm-lock.yaml

```diff
@@ -1,8 +1,4 @@
-lockfileVersion: '6.1'
```

Comment:
@Shazil2154  add this line in .gitignore
`pnpm-lock.yaml`
`**/pnpm-lock.yaml`
then delete this pnpm-lock.yaml file and push

Reply:
since we all are using different versions of pnpm this will cause issue. That is my bad but let's fix it in this pr

Reply:
sure

Reply:
this part is already handled in this pr #13